models/payment.py

```diff
@@ -295,19 +295,39 @@ def manual_refund(self):
 
         super(BankPayment, self).manual_refund()
 
+    @property
+    def recommended_destination(self):
+        for currency in [self.currency, "GBP"]:
+            try:
+                return BankAccount.query.filter_by(currency=currency, active=True).one()
+            except:
+                continue
+
 
 class BankAccount(db.Model):
     __tablename__ = "bank_account"
     __export_data__ = False
     id = db.Column(db.Integer, primary_key=True)
-    sort_code = db.Column(db.String, nullable=False)
-    acct_id = db.Column(db.String, nullable=False)
+    sort_code = db.Column(db.String)
+    acct_id = db.Column(db.String)
     currency = db.Column(db.String, nullable=False)
-
-    def __init__(self, sort_code, acct_id, currency="GBP"):
+    active = db.Column(db.Boolean)
+    institution = db.Column(db.String, nullable=False)
+    address = db.Column(db.String, nullable=False)
+    swift = db.Column(db.String, nullable=False)
+    iban = db.Column(db.String, nullable=False)
```

Comment:
I think it would probably be best to make SWIFT and IBAN nullable - although our existing Barclays GBP account does technically have an IBAN, it's not something we'd ever expect to use (it causes expense if someone attempts to send euros to it), so it's probably better if we don't worry about it.

I think this would also slightly simplify the migration.